homeworks/src/main/java/hw40/entity/OrderProduct.java

```diff
@@ -0,0 +1,30 @@
+package hw40.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+@Entity
+@Table(name = "order_product", schema = "postgres")
+public class OrderProduct {
```

Comment:
Why do you need to specify this table? It is primarily for many-to-many. Many-to-one relationships can be done via 2 tables and  it should not cause a problem